CONTRIBUTING.md

```diff
@@ -3,12 +3,16 @@
 Contributions to this Module are very welcome! We follow a fairly standard [pull request 
 process](https://help.github.com/articles/about-pull-requests/) for contributions, subject to the following guidelines:
  
-1. [File a GitHub issue](#file-a-github-issue)
-1. [Update the documentation](#update-the-documentation)
-1. [Update the tests](#update-the-tests)
-1. [Update the code](#update-the-code)
-1. [Create a pull request](#create-a-pull-request)
-1. [Merge and release](#merge-and-release)
+- [Contribution Guidelines](#contribution-guidelines)
+  - [File a GitHub issue](#file-a-github-issue)
+  - [Update the documentation](#update-the-documentation)
+  - [Update the tests](#update-the-tests)
+  - [Update the code](#update-the-code)
+    - [Backwards compatibility](#backwards-compatibility)
+    - [Downtime](#downtime)
+  - [Create a pull request](#create-a-pull-request)
+  - [Sign HashiCorp CLA](#sign-hashicorp-cla)
+  - [Merge and release](#merge-and-release)
```

Comment:
These seem to have an extra layer of indentation that isn't necessary

Reply:
My editor was automatically forammting and adding this

Reply:
NIT: Looks like the numbers (`1.`) were replaced with dashes (`-`) unnecessarily.